doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -48,13 +47,13 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.inf.com.01` | Compute | The Architecture shall/must xxx. |
+| `req.inf.com.01` | Compute | The Architecture **must** provide compute resources for VM Instancs. |
 | `req.inf.com.02` | Compute | The Architecture shall/must xxx. |
-| `req.inf.stg.01` | Storage | The Architecture **must** provide storage solution for VIM Instances and VNFs to support Block/Image and local VNF File system storage. |
+| `req.inf.stg.01` | Storage | The Architecture **must** provide storage solution for VM Instances to support Block/Image and local VNF File system storage. |
```

Comment:
Isn't the local VNF file system storage is a block storage used by the VNF?

Reply:
@CsatariGergely Would the change to "The Architecture **must** provide Block, Object and File system storage solutions for VM Instances." work?

Personally, these are 3 requirements.

Reply:
I would leave out the file system storage part. If the infra provides the file system storage that means that the application needs to support the fs supported by the infra, which is an extra dependency.

Reply:
@CsatariGergely what would be your proposed change?

Reply:
"The Architecture must provide Block, and Object storage solutions for VM Instances."  or "The Architecture must provide Block, Object and may provide File system storage solutions for VM Instances.".

Reply:
Thanks @CsatariGergely change reflected.